solara/website/pages/examples/visualization/annotator.py

```diff
@@ -0,0 +1,57 @@
+"""# Image annotation with Solara
+
+This example displays how to annotate images with different drawing tools in plotly figures. Use the canvas 
+below to draw shapes and visualize the canvas callback.
+
+
+Check [plotly docs](https://dash.plotly.com/annotations) for more information about image annotation.
+"""
+import json
+import plotly.graph_objects as go
+
+import solara
+
+title = "Plotly Image Annotator"
+text = solara.reactive('Draw on canvas')
+
+class CustomEncoder(json.JSONEncoder):
+    def default(self, o):
+        # Convert object instances to their string representation
+        if isinstance(o, object):
+            return str(o)
+        return super().default(o)
+
+@solara.component
+def Page():
+    def on_relayout(data):
+        if data is None:
+            return
+
+        relayout_data = data['relayout_data']
+
+        if "shapes" in relayout_data:
+            text.value = str(json.dumps(relayout_data["shapes"], indent=2, cls=CustomEncoder))
+
+    with solara.Div() as main:
```

Comment:
I think you can remove the Div, unless you added it for a reason. and unindent the block containing it.